data/json/items/melee/misc.json

```diff
@@ -34,7 +35,8 @@
   },
   {
     "id": "lobotomizer",
-    "type": "TOOL",
+    "type": "GENERIC",
+    "category": "weapons",
     "name": { "str": "lobotomizer" },
     "description": "This is a hand-forged collapsible tool that has two axe heads and sharp shovel-like tip on one end.  It can be used as a shovel, or you could chop some zombies with it instead.",
```

Comment:
```suggestion
    "description": "This is a hand-forged collapsible tool that has two axe heads and a sharp shovel-like tip on one end.  It can be used as a shovel, or you could chop some zombies with it instead.",
```